app/controllers/coronavirus_form/contact_details_controller.rb

```diff
@@ -1,13 +1,10 @@
 # frozen_string_literal: true
 
 class CoronavirusForm::ContactDetailsController < ApplicationController
-  include ActionView::Helpers::SanitizeHelper
-  include FieldValidationHelper
-  include FormFlowHelper
-
   before_action :check_first_question_answered, only: :show
 
   REQUIRED_FIELDS = %w(contact_name phone_number).freeze
+  TEXT_FIELDS = %w(contact_name role phone_number email).freeze
```

Comment:
Watch out for symbol vs string clashes - this caught us out a bit in the vulnerable people form?

Reply:
Yeah, this doesn't seem to have been a problem here

Reply:
I think this checks out because you're using the params:
https://github.com/alphagov/govuk-coronavirus-business-volunteer-form/pull/106#discussion_r397735176